frontend/src/components/ViewActivities/activityfns.js

```diff
@@ -127,3 +127,39 @@ export function getRefValue(ref, ignoreValue='', defaultValue=null) {
   } 
   return ref.current.value;
 }
+  
+/**
+ * Pretty format start and end times/days.
+ * 
```

Comment:
May be worth giving an example or two with some of the different types of formatting based on the inputs? Up to you.